### DISCUSSION TITLE
osu!lazer doesn't seem to be able to play audio correctly for using OpenGL renderer.

### DISCUSSION THREAD

**Discussion by tematikkp** [ppy/osu#23255] [Q&A]
### Type

Performance

### Bug description

When I first opened the game after starting up my computer, I saw this error, which is the screenshot that I'm showing here.

Also, not only this error, but also many other bugs when using OpenGL in this version which had never happened before, which are OpenTabletDriver stopping at some point, causing a bug. It was surely an unpleasant experience when I'm trying to boost my FPS using other renderer, which is this one.

### Screenshots or videos

https://ibb.co/1ZfnCnd

### Version

2023.419.0-lazer

### Logs

[legacy-ipc.log](https://github.com/ppy/osu/files/11283900/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11283901/network.log)
[performance.log](https://github.com/ppy/osu/files/11283902/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11283903/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11283904/updater.log)
[database.log](https://github.com/ppy/osu/files/11283905/database.log)
[input.log](https://github.com/ppy/osu/files/11283906/input.log)

**Comment by frenzibyte** [comment]
That notification is a sign that the intro screen spent too long to the point osu! thought it's not progressing at all due to audio issues, while in reality it's happening because of the rendering performance. Have you tried using Direct3D 11? Also interested to know how OpenGL (Legacy) performs for you.

**Reply by tematikkp** [reply]
If I use Direct3D 11, the average fps would be displayed as 60fps (the fps is locked to 60fps instead of higher fps). So, I'm using the legacy OpenGL renderer (it has more FPS than Direct3D), and I don't care if it has some latency (it's unable to run in exclusive fullscreen). My computer uses integrated graphics.

Not only that, but I experienced lag spikes while using OpenGL because it uses much RAM, and my OpenTabletDriver crashed because of that. (https://github.com/ppy/osu/issues/23252)

**Reply by peppy** [reply]
Please continue to use legacy gl or directx, not the new opengl renderer for now.
